backend/src/main/java/de/neuefische/backend/controller/RandMController.java

```diff
@@ -0,0 +1,29 @@
+package de.neuefische.backend.controller;
+
+import lombok.RequiredArgsConstructor;
+import de.neuefische.backend.model.RandMCharacter;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import de.neuefische.backend.service.RandMService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/randm")
+@RequiredArgsConstructor
+public class RandMController {
+    private final RandMService randMService;
+
+    @GetMapping("/characters")
+    public List<RandMCharacter> getAllCharacters(){
+        return randMService.getAllCharacters();
+    }
+
+    @GetMapping("/fill")
```

Comment:
Do you really need this endpoint?

Reply:
/fill endpoint is required for future DB connection, since it'd be inconvenient to keep sending requests for every user. 
This specific endpoint will fill our not yet implemented DB where the frontend will receive data from.